### DISCUSSION TITLE
Debug Leptos Application

### DISCUSSION THREAD

**Discussion by thanhdatvo** [leptos-rs/leptos#1390] [General]
I'm using 
```
cargo leptos watch
```
to run Leptos Application.
Is this possible to debug Leptos server or client with break point?

**Comment by aperepel** [comment]
I'm curious to know if there's anything better than the still-maturing ecosystem of today: https://rustwasm.github.io/docs/book/reference/debugging.html

**Reply by aperepel** [reply]
This is the most recent I could find. Doable, it seems, but with extra hops. DX can be improved, for sure, but this is a very dynamic field at the moment, so who knows, in a year or two? :D https://users.rust-lang.org/t/getting-raw-wasm-debugging-working-nicely-in-chrome-devtools/94646
